cmd/deploy.go

```diff
@@ -38,16 +41,24 @@ type deployCommandConfig struct {
 }
 
 type AppsodyApplication struct {
-	APIVersion string   `yaml:"apiVersion"`
-	Kind       string   `yaml:"kind"`
-	Metadata   Metadata `yaml:"metadata"`
-	Spec       Spec     `yaml:"spec"`
+	APIVersion string                         `yaml:"apiVersion" json:"apiVersion"`
+	Kind       string                         `yaml:"kind" json:"kind"`
+	Metadata   Metadata                       `yaml:"metadata" json:"metadata"`
+	Spec       v1beta1.AppsodyApplicationSpec `yaml:"spec" json:"spec"`
 }
 type Metadata struct {
-	Name string `yaml:"name"`
+	Name        string            `yaml:"name" json:"name"`
+	Labels      map[string]string `yaml:"labels" json:"labels"`
+	Annotations map[string]string `yaml:"annotations" json:"annotations"`
 }
-type Spec struct {
-	ApplicationImage string `yaml:"applicationImage"`
+
+var supportedLabels = []string{
```

Comment:
From the code I see that these will become k8s resource "labels" and everything else will be "annotations". But I didn't know what this meant until I dug into the code.  A better variable name or comment would be nice here.